src/styles/stars.scss

```diff
@@ -0,0 +1,30 @@
+.stars {
+  @include flexbox-between;
+  margin-bottom: 24px;
+
+  &__4 {
+    display: flex;
+  }
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    height: 16px;
+    width: 16px;
+    border-radius: 0.5px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+
+    &:nth-child(-n+4) {
+      background: url(../images/star-active.svg) center no-repeat;
+    }
+  }
+
+  &__review {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $font-color-dark;
+  }
```

Comment:
`review` should be `card blocks` element, not `stars`. This way would be easer to reuse `stars` block in case which we have no comment, or another name of it.

Reply:
++